test/cell_test.rb

```diff
@@ -0,0 +1,81 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/cell'
+
+class CellTest < Minitest::Test
+
+  def setup
+    @cell = Cell.new("B4")
+  end
+
+  def test_it_exists
+    assert_instance_of Cell, @cell
+  end
+
+  def test_it_has_a_coordinate
+    assert_equal "B4", @cell.coordinate
+  end
+
+  def test_cell_is_empty
+    assert_equal nil, @cell.ship
```

Comment:
I think we want this to be a separate test:

def test_ship_placement
  assert_equal nil, @cell.ship . #initial state#
  
  @cell.place_ship(ship_name) . # I think this is the right method that will say how/where the ship is placed on he board.#
  assert_equal ship_name, @cell.ship
end